singer/messages.py

```diff
@@ -189,6 +189,7 @@ def parse_message(msg):
         time_extracted = obj.get('time_extracted')
         if time_extracted:
             try:
+                # pylint: disable=c-extension-no-member
```

Comment:
Can you try to add `--extension-pkg-whitelist=ciso8601` to the pylint command instead?

Reply:
thank you fixed by 0f36fa9e9ab18d89cabbd4a9612f8b54c271f36b